builtin/providers/aws/resource_aws_placement_group.go

```diff
@@ -0,0 +1,150 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsPlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsPlacementGroupCreate,
+		Read:   resourceAwsPlacementGroupRead,
+		Delete: resourceAwsPlacementGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"strategy": &schema.Schema{
```

Comment:
I think we should set a default value of `Cluster` here as it is the **only** supported value atm